include/retdec/fileformat/file_format/elf/elf_format.h

```diff
@@ -23,6 +24,11 @@ namespace fileformat {
 class ElfFormat : public FileFormat
 {
 	private:
+		std::vector<std::string> telfhashSymbols;
```

Comment:
Does this have to be in the header? It would be best to have it only locally in source and pass it to `loadTelfhash()` fuction.